lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/validation/Gpg45F2fValidator.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.di.ipv.core.library.domain.gpg45.validation;
+
+import uk.gov.di.ipv.core.library.domain.gpg45.domain.CredentialEvidenceItem;
+
+public class Gpg45F2fValidator {
+
+    private Gpg45F2fValidator() {
+        throw new IllegalStateException("Utility class");
```

Comment:
An empty constructor would be fine and would matche up with other static classes.